aws-enigma-2/main.tf

```diff
@@ -0,0 +1,205 @@
+  default = "travisci/ci-garnet:packer-1512502276-986baf0"
+}
+
+terraform {
+  backend "s3" {
+    bucket         = "travis-terraform-state"
+    key            = "terraform-config/aws-enigma-2.tfstate"
+    region         = "us-east-1"
+    encrypt        = "true"
+    dynamodb_table = "travis-terraform-state"
+  }
+}
+
+provider "aws" {
+  version = "~> 1.4"
+}
+
+provider "heroku" {
+  version = "0.1.0"
+}
+
+data "terraform_remote_state" "vpc" {
+  backend = "s3"
+
+  config {
+    bucket         = "travis-terraform-state"
+    key            = "terraform-config/aws-shared-2.tfstate"
+    region         = "us-east-1"
+    dynamodb_table = "travis-terraform-state"
+  }
+}
+
+resource "random_id" "cyclist_token_org" {
+  byte_length = 32
+}
+
+module "rabbitmq_worker_config_org" {
+  source         = "../modules/rabbitmq_user"
+  admin_password = "${var.rabbitmq_password_org}"
+  admin_username = "${var.rabbitmq_username_org}"
+  endpoint       = "https://${trimspace(file("${path.module}/config/CLOUDAMQP_URL_HOST_ORG"))}"
+  scheme         = "${trimspace(file("${path.module}/config/CLOUDAMQP_URL_SCHEME_ORG"))}"
+  username       = "travis-worker-ec2-${var.env}-${var.index}"
+  vhost          = "${replace(trimspace("${file("${path.module}/config/CLOUDAMQP_URL_PATH_ORG")}"), "/^//", "")}"
+}
+
+data "template_file" "worker_config_org" {
+  template = <<EOF
+### config/worker-org-local.env
+${file("${path.module}/config/worker-org-local.env")}
+### config/worker-org.env
+${file("${path.module}/config/worker-org.env")}
+### worker.env
+${file("${path.module}/worker.env")}
+
+export TRAVIS_WORKER_AMQP_URI=${module.rabbitmq_worker_config_org.uri}
+export TRAVIS_WORKER_HARD_TIMEOUT=50m0s
+export TRAVIS_WORKER_TRAVIS_SITE=org
+export TRAVIS_WORKER_DOCKER_INSPECT_INTERVAL=1000ms
+EOF
+}
+
+module "aws_az_1b" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1b"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1b_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_az_1b2" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1b2"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1b_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_az_1e" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1e"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1e_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_az_1e2" {
```

Comment:
I think that the `aws_az_1*2` modules can be dropped.  Maybe even trim down to a single AZ?